### ISSUE TITLE
[A11Y] Missing "skip to..." link element at top /header region

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#9312]
## Issue Description
// What is the nature of the issue? How does it impact users?

There isn't a quick/easy way to start a new submission if you don't use a mouse.  Currently, it requires ~26 tabbing moves to reach the "new submission" and start the process if you are an editor and about ~8  tabbing moves if you are an author. Users enrolled as editors have to tab through the whole page to reach the list of submissions which increases even more the amount of "tabbings".

## How to reproduce / Current behavior
// Step-by-step instructions explaining how to reproduce the problem. The environment required, explicit identification of the component in question

Log in as an editor/author
1. Using the tab key on the keyboard, start to press it to move through focusable elements 
2. Try to reach the "New Submission" button only using the TAB key.
3. Count the amount of times it has been pressed

## What application are you using?
 OJS, OMP or OPS version X.X.X

OJS 3.4

## Stack used 
- Device: laptop
- OS: MacOS (Monterey)
- Browser: Chrome 
- Version: 117

## Testing method
- Manual (keyboard)
- Automated test: Accessibility Insights

## Action
// Describe action that can be taken to remediate the issue. 

Implement "skip to content/menu/submission list"  links as the first tabbing navigation element on pages.

## Additional information
// Please add any screenshots, code snippets, or other information we can use to investigate this issue.

![Image](https://github.com/pkp/pkp-lib/assets/4992902/10d62a8c-5ba4-4880-bffe-03932bceaaa9)

**Comment by jardakotesovec** [comment]
Addressed in 3.5.
